### DISCUSSION TITLE
How many merged pull requests do I need to get write access?

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#754] [(A) Anything]
@PatheticMustan 
@ArcerionDev

**Comment by NX721** [comment]
Be a verified coder and a contributor like me.

**Comment by NX721** [comment]
@hellonearth311.

**Comment by ghost** [comment]
depends.

**Comment by ghost** [comment]
i've been contributing for over a year, recently got maintain access

**Comment by NX721** [comment]
I'm also contributing, so I got the role in Discord.

**Comment by hellonearth311** [comment]
hmm, so might take a few months for me to get write acess?

**Reply by ghost** [reply]
yes and dedication and proving you're trustworthy and formality etc

**Reply by NX721** [reply]
> hmm, so might take a few months for me to get write acess?

I've been contributing for a while now, so I get the role in Discord and beside my name on the GitHub.

**Reply by ghost** [reply]
he means the ability to directly edit files and stuff

btw nx721 the pmgh staff say you're a skid kek

**Reply by NX721** [reply]
> hmm, so might take a few months for me to get write acess?

Ya gotta experiment and see if the code works and make it into a pull request see the ones that are closed with my name for details.
Imma be AFK fyi c ya

**Reply by NX721** [reply]
> he means the ability to directly edit files and stuff
> 
> btw nx721 the pmgh staff say you're a skid kek

Mk

**Comment by PatheticMustan** [comment]
We're not looking for new contributors at the moment, but if we were, you would require a few things:
- a discord account (pretty basic)
- time
- Actual contributions, not just random README fixes and small updates. This usually means legitimately fixing issues.


If you want to find things to do, you can look in the milestones. Right now we're on milestone 2.1.0, so those would be a pretty good indicator of skill.

**Comment by NX721** [comment]
@hellonearth311  What is your Discord Tag and Username?

**Reply by hellonearth311** [reply]
i dont have a discord account, but i'll make 1 today and tell u ASAP.

**Reply by NX721** [reply]
Ok Once you do DM me at 4:00 PM.
